releaseNotes/AzureDevOpsPushPipelineExtension.md

```diff
@@ -2,6 +2,15 @@
 Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version information can be found in the console when the task is executed:
 ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
 
+### v1.1.4 - December, 10 2020
+* Updated the readme 
+
+### v1.1.3 - December, 10 2020
+* Erorr handling in the case where, required parameters are missing 
+
+### v1.0.4 - November, 11 2020
+* Fixed an issue causing the task to fail [#431](https://github.com/Azure/AppConfiguration/issues/431)

```

Comment:
The description for this release is odd because we don't have any documentation for 1.0.3 which introduced the failure. So from the release notes it looks like we fixed a bug that existed since 1.0.0 preview. I think we could do better here and say "Reverted all changes in v1.0.3 to hotfix an issue causing task failures." to make the notes more continuous.